assets/js/modules/tagmanager/components/settings/SettingsView.js

```diff
@@ -263,6 +264,7 @@ export default function SettingsView() {
 						</P>
 					) }
 				</div>
+				<SettingsGoogleTagGatewayStatus />
 			</div>
```

Comment:
Let's use the `SettingsStatuses` component. The GTG status must be displayed in a separate row, as it is unrelated to the "Tag Manager Code Snippet".
```suggestion
			</div>
			<SettingsStatuses
				statuses={ [
					{
						label: __(
							'Google tag gateway for advertisers',
							'google-site-kit'
						),
						status: googleTagGatewayEnabled,
					},
				] }
			/>
```

<img width="603" height="324" alt="Screenshot 2025-09-08 at 1 27 01 PM" src="https://github.com/user-attachments/assets/a17984ff-b96b-421a-a971-2fa4eb19f9ce" />